sycl/include/syclcompat/syclcompat.hpp

```diff
@@ -29,7 +29,6 @@
 #include <syclcompat/id_query.hpp>
 #include <syclcompat/kernel.hpp>
 #include <syclcompat/launch.hpp>
-#include <syclcompat/launch_policy.hpp>
```

Comment:
Nit: If `syclcompat/launch_policy.hpp` is not meant to be included directly by the user, but through `launch.hpp`, should it maybe live in a different path? (`syclcompat/launch/launch_policy.hpp`, or something like that)

Reply:
Made a ticket to explore this before 2025.1, as discussed :+1: